src/components/Resume/Stars.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react'
+import starsfilled from '../../icons/star-fill.svg'
+import starsblank from '../../icons/star-blank.svg'
+
+const Stars = ({ lev }) => {
+    const stars = [];
+    for (let i = 1; i <= level(lev); i++) {
+        stars.push(<img src={starsfilled} key={i} alt="filled-star" />)
+    }
+    for (let i = level(lev) + 1; i <= 5; i++) {
+        stars.push(<img src={starsblank} key={i} alt="blank-star" />)
+    }
+    return (
+        <div className="keyword-level">
+            {stars}
+        </div>
+    );
+}
+
+function level (lev) {
```

Comment:
Please follow the `eslint` coding style from the project. Spacing rules and prefer arrow functions.